html and css front end changes

cjrhansen2 opened the pull request.

cjrhansen2 committed:
html and css front end changes

breakfastitem reviewed (changes requested):
Can you add a br line on parties template so that each new party appears on a new line. 
Can we add mabye 5 more movies to the list, and display the movies responsively.
When the navbar is expanded the button to collapse the navbar disappears.

cjrhansen2 committed:
fixed what I could and added movies

cjrhansen2 commented:
I added a <br> tag, added a few more movies, and fixed the background so it wasnt stretched out to fit the size of the list. I can't figure out how to make the movies wrap horizontally though.

breakfastitem committed:
Changed movie rows to be 2 by 2
